xapian-applications/omega/templates/query

```diff
@@ -153,3 +153,4 @@ $map{$cgilist{B},<input type=hidden name=B value="$html{$_}">}
 </body>
 </html>
 $log{query.log}
+$log{search.log,$hash{md5,$transform{s/^\s+|\s+$$//g,\1,$query}}$.$csv{$query,1}$.$csv{$hitlist{$id$if{$ne{$hit,$sub{$last,1}},$.}},1}$.$topdoc}
```

Comment:
This logs unconditionally.

It also normalises the query string and calculates its hash a second time, which is (a) inefficient and (b) just asking for someone to change one version but not the other and then we end up with the query ids in `search.log` and `clicks.log` not matching.